react/features/large-video/components/LargeVideo.web.js

```diff
@@ -70,6 +73,16 @@ export default class LargeVideo extends Component {
                 <span id = 'localConnectionMessage' />
                 { interfaceConfig.filmStripOnly ? null : <VideoQualityLabel /> }
                 <RecordingLabel />
+                <TranscriptionSubtitles />
```

Comment:
We never check if the transcriptions are actually enabled before adding this component. The RecordingLabel above has an isVisible check in its render method for example, but I can't see anything for the TranscriptionSubtitles.